### ISSUE TITLE
Generate confidence intervals for `predict`

### ISSUE THREAD

**Issue by jariji** [FixedEffects/FixedEffectModels.jl#213]
I would like to have confidence intervals for `predict`. In GLM.jl, `predict` takes a keyword argument `interval=:confidence` that permits this. Is it possible to get confidence intervals in FixedEffectModels predictions?

**Comment by matthieugomez** [comment]
Yes, it should not be too hard to do — you're welcome to try adding it if you have some time. Here is the code to modify:
https://github.com/FixedEffects/FixedEffectModels.jl/blob/eb7bf7318f65841446821c4b7a82820a00b803fc/src/FixedEffectModel.jl#L75
